### ISSUE TITLE
'Reset All' doesn't reset the auto-populated routes

### ISSUE THREAD

**Issue by dragon12** [tobymao/18xx#607]
I need to click 'Reset Train' on each train to reset to nothing, Reset All doesn't appear to affect them if they've been auto-populated.

**Comment by gchambers419** [comment]
Reset all returns all routes to the values from the previous run (there is no auto-calculate yet)
It's useful if you think there's a more profitable run to be made but you don't find one and decide to repeat the last run after all.

**Comment by dragon12** [comment]
Right, I understand what it's doing I just don't think that should be what it does :-) Reset Train resets to nothing, Reset All should reset all trains to nothing

**Comment by jenf** [comment]
There should be two buttons 'Clear All' and 'Reset All'
probably rephrase Reset Train as 'Clear Train'
